src/account_processing.yml

```diff
@@ -172,7 +172,22 @@ Resources:
             Bucket:
               Ref: ADFAccountBucket
             Events: s3:ObjectCreated:*
-
+            Filter:
+              S3Key:
+                Rules:
+                - Name: suffix
+                  Value: yml
+        S3Event2:
```

Comment:
Can these have better names that just S3Event and S3Event2?

Reply:
What about S3YamlSuffixEvents and S3YmlSuffixEvents?

Reply:
Works for me!

Reply:
Done